core/action.class.inc.php

```diff
@@ -877,4 +917,13 @@ public function DoCheckToWrite()
 			}
 		}
 	}
+
+	/**
+	 * @inheritDoc
+	 * @since 3.2.0
+	 */
+	public function GetAsynchronousGlobalSetting(): bool
+	{
+		return MetaModel::GetConfig()->Get('email_asynchronous');
```

Comment:
```suggestion
		return utils::GetConfig()->Get('email_asynchronous');
```

We rather use the `utils::GetConfig()` pattern instead as in some cases the metamodel isn't loaded. even though it is the case here, one could copy / paste this pattern elsewhere.